### PR TITLE
HUGE performance increase

### PR DESCRIPTION
**Pull request by justinmuskopf**

### CODE REVIEW

**Comment by sarahsturgeon** [comment]
can I get a deep analysis on the difference? 1.5M executions average with min/max should be fine.

File: lua/autorun/server/sv_model_regulator.lua [hunk 1]
```diff
@@ -9,7 +9,7 @@ local entityMeta = FindMetaTable("Entity")
 local setModel = entityMeta.SetModel
 
 function playerMeta:SetModel(desiredModel)
-	if modelIsProhibited[desiredModel] then return setModel(self, defaultModel) end
+	local model = modelIsProhibited[desiredModel] and defaultModel or desiredModel end
```
Comment (sarahsturgeon) [inline lua/autorun/server/sv_model_regulator.lua]:
```suggestion
local model = ( modelIsProhibited[desiredModel] and defaultModel ) or desiredModel end
```

you madman

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]

File: lua/autorun/server/sv_model_regulator.lua (hunk 1 above)
Comment (justinmuskopf) [reply lua/autorun/server/sv_model_regulator.lua]: this just the way LUA ternaries
http://lua-users.org/wiki/TernaryOperator

Comment (sarahsturgeon) [reply lua/autorun/server/sv_model_regulator.lua]: I just think it's clearer with the parens, that's all. Should still function the same.

**Pull request merged**